codebuild/scripts/build_deploy_integration_container.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
nit: Can add a quick overview comment? The name of this file is good but a few sentences explaining purpose will help new developers understand more quickly.